### PR TITLE
Added test groups

### PR DESCRIPTION
**Pull request by grigory-bogush**
https://trello.com/c/zsMQtF74/40-xxx

### CODE REVIEW

File: app/controllers/test_groups_controller.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+class TestGroupsController < ApplicationController
+  before_action :set_test_group, only: [:show, :edit, :update, :destroy]
```
Comment (DmitryLeviossa) [inline app/controllers/test_groups_controller.rb:2]: only: %i[show edit update destroy]

File: app/controllers/test_groups_controller.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    if @test_group.update(test_group_params)
+      redirect_to @test_group, notice: 'Test group was successfully updated.'
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    @test_group.destroy
+    redirect_to test_groups_url, notice: 'Test group was successfully destroyed.'
```
Comment (DmitryLeviossa) [inline app/controllers/test_groups_controller.rb:35]: test_groups_path

File: app/models/test_group.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class TestGroup < ApplicationRecord
+  belongs_to :company
+  has_many :tests
```
Comment (DmitryLeviossa) [inline app/models/test_group.rb:3]: add uniq validation for name by company_id

File: app/views/test_groups/edit.slim [hunk 4]
```diff
@@ -0,0 +1,8 @@
+= render 'layouts/form' do
+  .new-form
+    .new-form--content
+      .new-form--content--title
+        h1 Editing test group
+      .new-form--content--form
+        = render 'form', test_group: @test_group
+        .new-form--content--form--image.test-img
```
Comment (DmitryLeviossa) [inline app/views/test_groups/edit.slim:8]: Why do we need that div here?

File: db/migrate/20200911140139_create_test_groups.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
+class CreateTestGroups < ActiveRecord::Migration[5.2]
+  def change
+    create_table :test_groups do |t|
+      t.string :name, null: false
+      t.references :company, index: true, foreign_key: true
+
+      t.timestamps
+    end
+
+    add_reference :tests, :test_group, index: true, foreign_key: true
```
Comment (DmitryLeviossa) [inline db/migrate/20200911140139_create_test_groups.rb:10]: Columns for tests
Add please value int column, by default it's 0
Add please index column, by default it's 1
Add description column as text
add uniq indexes for DB on test_group and name | name and index

Comment (grigory-bogush) [reply db/migrate/20200911140139_create_test_groups.rb:10]: "index" to "module"

File: app/views/test_groups/edit.slim (hunk 4 above)
Comment (grigory-bogush) [reply app/views/test_groups/edit.slim:8]: It has an illustration as background-image, all the illustrations are done with background-image

Screenshot 2020-09-14 at 17 19 15

**Review by DmitryLeviossa — APPROVED** [review]

**Pull request merged**